app/controllers/api/v1/events_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::EventsController < ApplicationController
+  def index
+    render json: Event.all
```

Comment:
Logan, will you please add comments as you are writing to help me when I'm following along on my own. Thanks!

Reply:
Of course, I will start doing that now.